main.rb

```diff
@@ -0,0 +1,135 @@
+require './app'
+def create_student(app)
+  puts 'Let\'s create a student!'
+  puts 'What is your name?'
+  name = gets.chomp
+  puts 'What is your age?'
+  age = gets.chomp.to_i
+  puts 'Has parents permission? (y/n)'
+  parent_permission = gets.chomp.downcase == 'y'
+  app.create_student(age: age, name: name, parent_permission: parent_permission)
+  puts 'Student created!'
+end
+
+def create_teacher(app)
+  puts 'Let\'s create a teacher!'
+  puts 'What is your name?'
+  name = gets.chomp
+  puts 'What is your age?'
+  age = gets.chomp.to_i
+  puts 'What is your specialization?'
+  specialization = gets.chomp
+  app.create_teacher(specialization: specialization, age: age, name: name, parent_permission: true)
+  puts 'Teacher created!'
+end
```

Comment:
- [ ] When I am creating a teacher, I am getting the error below 👇 please fix it

![image](https://user-images.githubusercontent.com/75126481/174289734-122428e6-6bdc-48ee-9d15-76c22a00c831.png)